pkg/util/debugutil/debugutil.go

```diff
@@ -43,3 +46,9 @@ func init() {
 		return false
 	}(os.Getppid()))
 }
+
+// Stack wraps the output of debug.Stack() with redact.Safe() to avoid
+// unnecessary redaction.
+func Stack() interfaces.SafeValue {
```

Comment:
Given we've had issues in the past with high system CPU usage due to excessive use of debug.Stack in error handling pathways, we should probably add a warning comment here to the effect of "Calling debug.Stack grabs system-level locks and could cause high system CPU usage resulting in the Go runtime to lock up if called too frequently, even if called only in error-handling pathways. Use sporadically and only when necessary."

Reply:
Done
<!-- Reviewable comment -OCm-rTDAK-Y91DKqt3e:bb6un1j -->
<!-- Sent from Reviewable.io -->